### ISSUE TITLE
TaskSchedulerOnUnobservedTaskException on TakePhotoAsync

### ISSUE THREAD

**Issue by Obvious1992** [jamesmontemagno/MediaPlugin#239]
## Bug 
Version Number of Plugin: 2.6.2
Xam Forms Version: 2.3.3.193
Device Tested On: google Pixel/ S5
Simulator Tested On: Android Emulator for VS

### Expected Behavior
Camera should open to take a picture

### Actual Behavior
The camera doesn't open and the the method stops at TakePicture. When I popback on the Navigationstack I can see the following output:

05-16 18:09:06.961 E/Crash Report( 3736): Time: 5/16/2017 6:09:06 PM
05-16 18:09:06.961 E/Crash Report( 3736): Error: Unhandled Exception
05-16 18:09:06.961 E/Crash Report( 3736): System.Exception: TaskSchedulerOnUnobservedTaskException ---> System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. ---> Java.Lang.NullPointerException: Attempt to invoke virtual method 'android.content.res.XmlResourceParser android.content.pm.ProviderInfo.loadXmlMetaData(android.content.pm.PackageManager, java.lang.String)' on a null object reference
05-16 18:09:06.961 E/Crash Report( 3736):   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <d18287e1d683419a8ec3216fd78947b9>:0 
05-16 18:09:06.961 E/Crash Report( 3736):   at Java.Interop.JniEnvironment+StaticMethods.CallStaticObjectMethod (Java.Interop.JniObjectReference type, Java.Interop.JniMethodInfo method, Java.Interop.JniArgumentValue* args) [0x00069] in <5716a943049b49ca928a3a1c8d2386f4>:0 
05-16 18:09:06.961 E/Crash Report( 3736):   at Android.Runtime.JNIEnv.CallStaticObjectMethod (System.IntPtr jclass, System.IntPtr jmethod, Android.Runtime.JValue* parms) [0x0000e] in <20a5debecbfd420ba8345429bf71acbe>:0 
05-16 18:09:06.961 E/Crash Report( 3736):   at Android.Support.V4.Content.FileProvider.GetUriForFile (Android.Content.Context context, System.String authority, Java.IO.File file) [0x00078] in <27c17fe440cf491ba8255bcefade6e02>:0 
05-16 18:09:06.961 E/Crash Report( 3736):   at Plugin.Media.MediaPickerActivity.OnCreate (Android.OS.Bundle savedInstanceState) [0x00212] in C:\projects\mediaplugin\src\Media.Plugin.Android\MediaPickerActivity.cs:172 
05-16 18:09:06.961 E/Crash Report( 3736): --- End of stack trace from previous location where exception was thrown ---
05-16 18:09:06.961 E/Crash Report( 3736):   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <d18287e1d683419a8ec3216fd78947b9>:0 
05-16 18:09:06.961 E/Crash Report( 3736):   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) [0x0003e] in <d18287e1d683419a8ec3216fd78947b9>:0 
05-16 18:09:06.961 E/Crash Report( 3736):   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) [0x00028] in <d18287e1d683419a8ec3216fd78947b9>:0 
05-16 18:09:06.961 E/Crash Report( 3736):   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) [0x00008] in <d18287e1d683419a8ec3216fd78947b9>:0 
05-16 18:09:06.961 E/Crash Report( 3736):   at System.Runtime.CompilerServices.TaskAwaiter`1[TResult].GetResult () [0x00000] in <d18287e1d683419a8ec3216fd78947b9>:0 
05-16 18:09:06.961 E/Crash Report( 3736):   at Plugin.Media.MediaImplementation+<TakePhotoAsync>d__14.MoveNext () [0x000d0] in C:\projects\mediaplugin\src\Media.Plugin.Android\MediaImplementation.cs:119 
05-16 18:09:06.961 E/Crash Report( 3736): --- End of stack trace from previous location where exception was thrown ---
05-16 18:09:06.961 E/Crash Report( 3736):   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <d18287e1d683419a8ec3216fd78947b9>:0 
05-16 18:09:06.961 E/Crash Report( 3736):   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) [0x0003e] in <d18287e1d683419a8ec3216fd78947b9>:0 
05-16 18:09:06.961 E/Crash Report( 3736):   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) [0x00028] in <d18287e1d683419a8ec3216fd78947b9>:0 
05-16 18:09:06.961 E/Crash Report( 3736):   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) [0x00008] in <d18287e1d683419a8ec3216fd78947b9>:0 
05-16 18:09:06.961 E/Crash Report( 3736):   at System.Runtime.CompilerServices.TaskAwaiter`1[TResult].GetResult () [0x00000] in <d18287e1d683419a8ec3216fd78947b9>:0

I am not sure why this is happening, I was using this plugin for a long time without any problem. 

i appreciate any help
Julian

**Comment by jamesmontemagno** [comment]
What does your code look like?

**Comment by Obvious1992** [comment]
Hey James and thank you for your quick reply.
Here is my code:
`
            if (action == AppResources.NewImg)
            {
                //await Task.Delay(100);

                if (!CrossMedia.Current.IsCameraAvailable || !CrossMedia.Current.IsTakePhotoSupported)
                {
                    await DisplayAlert("No Camera", ":( No camera available.", "OK");
                    await LblAttachment.ScaleTo(1, 50, Easing.CubicIn);
                    return;
                }

                //get stucked here 
                var file = await CrossMedia.Current.TakePhotoAsync(new Plugin.Media.Abstractions.StoreCameraMediaOptions
                {
                    CompressionQuality = Convert.ToInt32(Settings.ImageScale),
                    PhotoSize = PhotoSize.Small,
                });

                if (file == null) { 
                    await LblAttachment.ScaleTo(1, 50, Easing.CubicIn);
                    return;
                }
}
`
Best regards
Julian

**Comment by jamesmontemagno** [comment]
You would need to create a small repo project for me to test, this simply isn't enough code for me to help out sorry.

**Comment by nikpodakov** [comment]
Hi James! I have the same issue (CrossMedia.Current.TakePhotoAsync throw Java.Lang.NullPointerException: Attempt to invoke virtual method 'android.content.res.XmlResourceParser android.content.pm.ProviderInfo.loadXmlMetaData(android.content.pm.PackageManager, java.lang.String)' on a null object reference), so I create small sample project that demonstrates it:
[https://bitbucket.org/nik_podakov/mediapluginsample/src](url)

**Comment by Obvious1992** [comment]
You were able to reproduce this error? Great!

**Comment by jamesmontemagno** [comment]
All looks good, what if you don't use an action sheet?

I have other fixes in newer versions for that, but mostly around iOS

**Comment by jamesmontemagno** [comment]
Also @nikpodakov  that looks like a different issue than what is here. So maybe open a new issue for it.

**Comment by nikpodakov** [comment]
I replaced action sheet on two buttons, and TakePhotoAsync still throws this exception (I updated the repo). I dont sure, but our stack traces looks very similar. There is my:
{Java.Lang.NullPointerException: Exception of type 'Java.Lang.NullPointerException' was thrown.
  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in /Users/builder/data/lanes/4468/f913a78a/source/mono/mcs/class/referencesource/mscorlib/system/runtime/exceptionservices/exceptionservicescommon.cs:143 
  at Java.Interop.JniEnvironment+StaticMethods.CallStaticObjectMethod (Java.Interop.JniObjectReference type, Java.Interop.JniMethodInfo method, Java.Interop.JniArgumentValue* args) [0x00082] in /Users/builder/data/lanes/4468/b16fb820/source/Java.Interop/src/Java.Interop/Java.Interop/JniEnvironment.g.cs:12649 
  at Android.Runtime.JNIEnv.CallStaticObjectMethod (System.IntPtr jclass, System.IntPtr jmethod, Android.Runtime.JValue* parms) [0x00000] in /Users/builder/data/lanes/4468/b16fb820/source/monodroid/src/Mono.Android/JNIEnv.g.cs:562 
  at Android.Support.V4.Content.FileProvider.GetUriForFile (Android.Content.Context context, System.String authority, Java.IO.File file) [0x00078] in <27c17fe440cf491ba8255bcefade6e02>:0 
  at Plugin.Media.MediaPickerActivity.OnCreate (Android.OS.Bundle savedInstanceState) [0x00212] in C:\projects\mediaplugin\src\Media.Plugin.Android\MediaPickerActivity.cs:172 
--- End of stack trace from previous location where exception was thrown ---
  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in /Users/builder/data/lanes/4468/f913a78a/source/mono/mcs/class/referencesource/mscorlib/system/runtime/exceptionservices/exceptionservicescommon.cs:143 
  at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) [0x00047] in /Users/builder/data/lanes/4468/f913a78a/source/mono/mcs/class/referencesource/mscorlib/system/runtime/compilerservices/TaskAwaiter.cs:187 
  at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) [0x0002e] in /Users/builder/data/lanes/4468/f913a78a/source/mono/mcs/class/referencesource/mscorlib/system/runtime/compilerservices/TaskAwaiter.cs:156 
  at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) [0x0000b] in /Users/builder/data/lanes/4468/f913a78a/source/mono/mcs/class/referencesource/mscorlib/system/runtime/compilerservices/TaskAwaiter.cs:128 
  at System.Runtime.CompilerServices.TaskAwaiter1[TResult].GetResult () [0x00000] in /Users/builder/data/lanes/4468/f913a78a/source/mono/mcs/class/referencesource/mscorlib/system/runtime/compilerservices/TaskAwaiter.cs:357 
  at Plugin.Media.MediaImplementation+<TakePhotoAsync>d__14.MoveNext () [0x000d0] in C:\projects\mediaplugin\src\Media.Plugin.Android\MediaImplementation.cs:119 
--- End of stack trace from previous location where exception was thrown ---
  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in /Users/builder/data/lanes/4468/f913a78a/source/mono/mcs/class/referencesource/mscorlib/system/runtime/exceptionservices/exceptionservicescommon.cs:143 
  at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) [0x00047] in /Users/builder/data/lanes/4468/f913a78a/source/mono/mcs/class/referencesource/mscorlib/system/runtime/compilerservices/TaskAwaiter.cs:187 
  at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) [0x0002e] in /Users/builder/data/lanes/4468/f913a78a/source/mono/mcs/class/referencesource/mscorlib/system/runtime/compilerservices/TaskAwaiter.cs:156 
  at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) [0x0000b] in /Users/builder/data/lanes/4468/f913a78a/source/mono/mcs/class/referencesource/mscorlib/system/runtime/compilerservices/TaskAwaiter.cs:128 
  at System.Runtime.CompilerServices.TaskAwaiter1[TResult].GetResult () [0x00000] in /Users/builder/data/lanes/4468/f913a78a/source/mono/mcs/class/referencesource/mscorlib/system/runtime/compilerservices/TaskAwaiter.cs:357 
  at MediaPluginSample.ViewModels.MainPageViewModel+<TakePhoto>d__14.MoveNext () [0x0031d] in D:\Repos\MediaPluginSample\MediaPluginSample\MediaPluginSample\MediaPluginSample\ViewModels\MainPageViewModel.cs:79 
  --- End of managed Java.Lang.NullPointerException stack trace ---
java.lang.NullPointerException
	at android.support.v4.content.FileProvider.parsePathStrategy(FileProvider.java:560)
	at android.support.v4.content.FileProvider.getPathStrategy(FileProvider.java:534)
	at android.support.v4.content.FileProvider.getUriForFile(FileProvider.java:376)
	at md5a3e164e78ade0c22cefea770ddd0bc49.MediaPickerActivity.n_onCreate(Native Method)
	at md5a3e164e78ade0c22cefea770ddd0bc49.MediaPickerActivity.onCreate(MediaPickerActivity.java:42)
	at android.app.Activity.performCreate(Activity.java:5231)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1087)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2148)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2233)
	at android.app.ActivityThread.access$800(ActivityThread.java:135)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1196)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:136)
	at android.app.ActivityThread.main(ActivityThread.java:5001)
	at java.lang.reflect.Method.invokeNative(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:515)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:785)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:601)
	at dalvik.system.NativeStart.main(Native Method)
}

**Comment by jamesmontemagno** [comment]
You will have to create a sample app for me to look at. It looks like you don't have a proper manifest.

**Comment by nikpodakov** [comment]
I've already created a sample app and sent link on it to you:
[https://bitbucket.org/nik_podakov/mediapluginsample/src](url)
And this is direct link on android manifest:
[https://bitbucket.org/nik_podakov/mediapluginsample/src/ce94dccb8b37f2195bd395c1bc3a549c38f7438f/MediaPluginSample/MediaPluginSample.Android/Properties/AndroidManifest.xml?at=master&fileviewer=file-view-default](url)
What is wrong with this sample app?

**Comment by jamesmontemagno** [comment]
So essentially here you have an invalid manifest.

When you set your package name to just: package="MediaPluginSample" the actual android manifest that gets generated creates: package="MediaPluginSample.MediaPluginSample"

you can find this in obj/Android/AndroidManifest.xml

Simply rename to a proper package name such as "com.test.mediasample"

**Comment by jamesmontemagno** [comment]
I have filed a bug with the team here: https://bugzilla.xamarin.com/show_bug.cgi?id=57819

Technically it looks like it can be a single string, but that is a terrible idea... https://developer.android.com/guide/topics/manifest/manifest-element.html#package

**Comment by nikpodakov** [comment]
Thank you a lot for answer.

**Comment by RAKESHRAGHAVAN** [comment]
My android app getting closed after taking image.Device Os versions and crash log are mentioned below

OS Version 
6.0.1
5.1


Caused by: android.runtime.JavaProxyThrowable: System.ObjectDisposedException: Cannot access a disposed object.
--
Object name: 'Android.Graphics.Bitmap'.
at Java.Interop.JniPeerMembers.AssertSelf (Java.Interop.IJavaPeerable self) [0x00029] in <5d8c9bafeaf046dc9177898a7b050df7>:0
at Java.Interop.JniPeerMembers+JniInstanceMethods.InvokeAbstractVoidMethod (System.String encodedMember, Java.Interop.IJavaPeerable self, Java.Interop.JniArgumentValue* parameters) [0x00000] in <5d8c9bafeaf046dc9177898a7b050df7>:0
at Android.Graphics.Bitmap.Recycle () [0x0000a] in <a1ff8381ed444e628d5466b87b31cb12>:0
at Xamarin.Forms.Platform.Android.ButtonDrawable.Reset () [0x00008] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.ButtonRenderer.OnElementChanged (Xamarin.Forms.Platform.Android.ElementChangedEventArgs`1[TElement] e) [0x0006c] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].SetElement (TElement element) [0x000fc] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].Xamarin.Forms.Platform.Android.IVisualElementRenderer.SetElement (Xamarin.Forms.VisualElement element) [0x00027] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementPackager.AddChild (Xamarin.Forms.VisualElement view, Xamarin.Forms.Platform.Android.IVisualElementRenderer oldRenderer, Xamarin.Forms.Platform.Android.RendererPool pool, System.Boolean sameChildren) [0x0003a] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementPackager.SetElement (Xamarin.Forms.VisualElement oldElement, Xamarin.Forms.VisualElement newElement) [0x0010d] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementPackager.<.ctor>b__6_0 (System.Object sender, Xamarin.Forms.Platform.Android.VisualElementChangedEventArgs args) [0x0000d] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].OnElementChanged (Xamarin.Forms.Platform.Android.ElementChangedEventArgs`1[TElement] e) [0x0002c] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].SetElement (TElement element) [0x000fc] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].Xamarin.Forms.Platform.Android.IVisualElementRenderer.SetElement (Xamarin.Forms.VisualElement element) [0x00027] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementPackager.AddChild (Xamarin.Forms.VisualElement view, Xamarin.Forms.Platform.Android.IVisualElementRenderer oldRenderer, Xamarin.Forms.Platform.Android.RendererPool pool, System.Boolean sameChildren) [0x0003a] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementPackager.SetElement (Xamarin.Forms.VisualElement oldElement, Xamarin.Forms.VisualElement newElement) [0x0010d] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementPackager.<.ctor>b__6_0 (System.Object sender, Xamarin.Forms.Platform.Android.VisualElementChangedEventArgs args) [0x0000d] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].OnElementChanged (Xamarin.Forms.Platform.Android.ElementChangedEventArgs`1[TElement] e) [0x0002c] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].SetElement (TElement element) [0x000fc] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].Xamarin.Forms.Platform.Android.IVisualElementRenderer.SetElement (Xamarin.Forms.VisualElement element) [0x00027] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementPackager.AddChild (Xamarin.Forms.VisualElement view, Xamarin.Forms.Platform.Android.IVisualElementRenderer oldRenderer, Xamarin.Forms.Platform.Android.RendererPool pool, System.Boolean sameChildren) [0x0003a] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementPackager.SetElement (Xamarin.Forms.VisualElement oldElement, Xamarin.Forms.VisualElement newElement) [0x0010d] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementPackager.<.ctor>b__6_0 (System.Object sender, Xamarin.Forms.Platform.Android.VisualElementChangedEventArgs args) [0x0000d] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].OnElementChanged (Xamarin.Forms.Platform.Android.ElementChangedEventArgs`1[TElement] e) [0x0002c] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].SetElement (TElement element) [0x000fc] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].Xamarin.Forms.Platform.Android.IVisualElementRenderer.SetElement (Xamarin.Forms.VisualElement element) [0x00027] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementPackager.AddChild (Xamarin.Forms.VisualElement view, Xamarin.Forms.Platform.Android.IVisualElementRenderer oldRenderer, Xamarin.Forms.Platform.Android.RendererPool pool, System.Boolean sameChildren) [0x0003a] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementPackager.SetElement (Xamarin.Forms.VisualElement oldElement, Xamarin.Forms.VisualElement newElement) [0x0010d] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementPackager.<.ctor>b__6_0 (System.Object sender, Xamarin.Forms.Platform.Android.VisualElementChangedEventArgs args) [0x0000d] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].OnElementChanged (Xamarin.Forms.Platform.Android.ElementChangedEventArgs`1[TElement] e) [0x0002c] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].SetElement (TElement element) [0x000fc] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].Xamarin.Forms.Platform.Android.IVisualElementRenderer.SetElement (Xamarin.Forms.VisualElement element) [0x00027] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementPackager.AddChild (Xamarin.Forms.VisualElement view, Xamarin.Forms.Platform.Android.IVisualElementRenderer oldRenderer, Xamarin.Forms.Platform.Android.RendererPool pool, System.Boolean sameChildren) [0x0003a] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementPackager.SetElement (Xamarin.Forms.VisualElement oldElement, Xamarin.Forms.VisualElement newElement) [0x0010d] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementPackager.<.ctor>b__6_0 (System.Object sender, Xamarin.Forms.Platform.Android.VisualElementChangedEventArgs args) [0x0000d] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].OnElementChanged (Xamarin.Forms.Platform.Android.ElementChangedEventArgs`1[TElement] e) [0x0002c] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].SetElement (TElement element) [0x000fc] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].Xamarin.Forms.Platform.Android.IVisualElementRenderer.SetElement (Xamarin.Forms.VisualElement element) [0x00027] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.ViewCellRenderer+ViewCellContainer.Update (Xamarin.Forms.ViewCell cell) [0x0009a] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.ViewCellRenderer.GetCellCore (Xamarin.Forms.Cell item, Android.Views.View convertView, Android.Views.ViewGroup parent, Android.Content.Context context) [0x00011] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.CellRenderer.GetCell (Xamarin.Forms.Cell item, Android.Views.View convertView, Android.Views.ViewGroup parent, Android.Content.Context context) [0x00057] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.CellFactory.GetCell (Xamarin.Forms.Cell item, Android.Views.View convertView, Android.Views.ViewGroup parent, Android.Content.Context context, Xamarin.Forms.View view) [0x00023] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.ListViewAdapter.GetView (System.Int32 position, Android.Views.View convertView, Android.Views.ViewGroup parent) [0x001ee] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Android.Widget.BaseAdapter.n_GetView_ILandroid_view_View_Landroid_view_ViewGroup_ (System.IntPtr jnienv, System.IntPtr native__this, System.Int32 position, System.IntPtr native_convertView, System.IntPtr native_parent) [0x00018] in <a1ff8381ed444e628d5466b87b31cb12>:0
at (wrapper dynamic-method) System.Object:3fbe8fb4-237b-42ce-a4d2-d4a12ab86b55 (intptr,intptr,int,intptr,intptr)
--- End of stack trace from previous location where exception was thrown ---
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <9ed750c09b8f45beba214b7bbb241a9e>:0
at Java.Interop.JniEnvironment+InstanceMethods.CallNonvirtualVoidMethod (Java.Interop.JniObjectReference instance, Java.Interop.JniObjectReference type, Java.Interop.JniMethodInfo method, Java.Interop.JniArgumentValue* args) [0x00089] in <5d8c9bafeaf046dc9177898a7b050df7>:0
at Java.Interop.JniPeerMembers+JniInstanceMethods.InvokeNonvirtualVoidMethod (System.String encodedMember, Java.Interop.IJavaPeerable self, Java.Interop.JniArgumentValue* parameters) [0x0001f] in <5d8c9bafeaf046dc9177898a7b050df7>:0
at Android.Views.ViewGroup.Layout (System.Int32 l, System.Int32 t, System.Int32 r, System.Int32 b) [0x00062] in <a1ff8381ed444e628d5466b87b31cb12>:0
at Xamarin.Forms.Platform.Android.ViewRenderer`2[TView,TNativeView].OnLayout (System.Boolean changed, System.Int32 l, System.Int32 t, System.Int32 r, System.Int32 b) [0x00059] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.ListViewRenderer.OnLayout (System.Boolean changed, System.Int32 l, System.Int32 t, System.Int32 r, System.Int32 b) [0x00000] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.FormsViewGroup.n_OnLayout_ZIIII (System.IntPtr jnienv, System.IntPtr native__this, System.Boolean p0, System.Int32 p1, System.Int32 p2, System.Int32 p3, System.Int32 p4) [0x00008] in <02fb1ab8b89c4df098f6cb3625e0c285>:0
at (wrapper dynamic-method) System.Object:d1f0d31b-ee80-4c16-b4e2-4cf698efa9e5 (intptr,intptr,bool,int,int,int,int)
--- End of stack trace from previous location where exception was thrown ---
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <9ed750c09b8f45beba214b7bbb241a9e>:0
at Java.Interop.JniEnvironment+InstanceMethods.CallNonvirtualVoidMethod (Java.Interop.JniObjectReference instance, Java.Interop.JniObjectReference type, Java.Interop.JniMethodInfo method, Java.Interop.JniArgumentValue* args) [0x00089] in <5d8c9bafeaf046dc9177898a7b050df7>:0
at Android.Runtime.JNIEnv.CallNonvirtualVoidMethod (System.IntPtr jobject, System.IntPtr jclass, System.IntPtr jmethod, Android.Runtime.JValue* parms) [0x00015] in <a1ff8381ed444e628d5466b87b31cb12>:0
at Xamarin.Forms.Platform.Android.FormsViewGroup.MeasureAndLayout (System.Int32 p0, System.Int32 p1, System.Int32 p2, System.Int32 p3, System.Int32 p4, System.Int32 p5) [0x000f6] in <02fb1ab8b89c4df098f6cb3625e0c285>:0
at Xamarin.Forms.Platform.Android.VisualElementTracker.UpdateLayout () [0x000bc] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].UpdateLayout () [0x0000e] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].OnLayout (System.Boolean changed, System.Int32 l, System.Int32 t, System.Int32 r, System.Int32 b) [0x0003d] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.FormsViewGroup.n_OnLayout_ZIIII (System.IntPtr jnienv, System.IntPtr native__this, System.Boolean p0, System.Int32 p1, System.Int32 p2, System.Int32 p3, System.Int32 p4) [0x00008] in <02fb1ab8b89c4df098f6cb3625e0c285>:0
at (wrapper dynamic-method) System.Object:d1f0d31b-ee80-4c16-b4e2-4cf698efa9e5 (intptr,intptr,bool,int,int,int,int)
--- End of stack trace from previous location where exception was thrown ---
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <9ed750c09b8f45beba214b7bbb241a9e>:0
at Java.Interop.JniEnvironment+InstanceMethods.CallNonvirtualVoidMethod (Java.Interop.JniObjectReference instance, Java.Interop.JniObjectReference type, Java.Interop.JniMethodInfo method, Java.Interop.JniArgumentValue* args) [0x00089] in <5d8c9bafeaf046dc9177898a7b050df7>:0
at Android.Runtime.JNIEnv.CallNonvirtualVoidMethod (System.IntPtr jobject, System.IntPtr jclass, System.IntPtr jmethod, Android.Runtime.JValue* parms) [0x00015] in <a1ff8381ed444e628d5466b87b31cb12>:0
at Xamarin.Forms.Platform.Android.FormsViewGroup.MeasureAndLayout (System.Int32 p0, System.Int32 p1, System.Int32 p2, System.Int32 p3, System.Int32 p4, System.Int32 p5) [0x000f6] in <02fb1ab8b89c4df098f6cb3625e0c285>:0
at Xamarin.Forms.Platform.Android.VisualElementTracker.UpdateLayout () [0x000bc] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].UpdateLayout () [0x0000e] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].OnLayout (System.Boolean changed, System.Int32 l, System.Int32 t, System.Int32 r, System.Int32 b) [0x0003d] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.FormsViewGroup.n_OnLayout_ZIIII (System.IntPtr jnienv, System.IntPtr native__this, System.Boolean p0, System.Int32 p1, System.Int32 p2, System.Int32 p3, System.Int32 p4) [0x00008] in <02fb1ab8b89c4df098f6cb3625e0c285>:0
at (wrapper dynamic-method) System.Object:d1f0d31b-ee80-4c16-b4e2-4cf698efa9e5 (intptr,intptr,bool,int,int,int,int)
--- End of stack trace from previous location where exception was thrown ---
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <9ed750c09b8f45beba214b7bbb241a9e>:0
at Java.Interop.JniEnvironment+InstanceMethods.CallNonvirtualVoidMethod (Java.Interop.JniObjectReference instance, Java.Interop.JniObjectReference type, Java.Interop.JniMethodInfo method, Java.Interop.JniArgumentValue* args) [0x00089] in <5d8c9bafeaf046dc9177898a7b050df7>:0
at Android.Runtime.JNIEnv.CallNonvirtualVoidMethod (System.IntPtr jobject, System.IntPtr jclass, System.IntPtr jmethod, Android.Runtime.JValue* parms) [0x00015] in <a1ff8381ed444e628d5466b87b31cb12>:0
at Xamarin.Forms.Platform.Android.FormsViewGroup.MeasureAndLayout (System.Int32 p0, System.Int32 p1, System.Int32 p2, System.Int32 p3, System.Int32 p4, System.Int32 p5) [0x000f6] in <02fb1ab8b89c4df098f6cb3625e0c285>:0
at Xamarin.Forms.Platform.Android.VisualElementTracker.UpdateLayout () [0x000bc] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].UpdateLayout () [0x0000e] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.PageContainer.OnLayout (System.Boolean changed, System.Int32 l, System.Int32 t, System.Int32 r, System.Int32 b) [0x00006] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Android.Views.ViewGroup.n_OnLayout_ZIIII (System.IntPtr jnienv, System.IntPtr native__this, System.Boolean changed, System.Int32 l, System.Int32 t, System.Int32 r, System.Int32 b) [0x00008] in <a1ff8381ed444e628d5466b87b31cb12>:0
at (wrapper dynamic-method) System.Object:e5b29d3a-037a-43b4-bc64-ad23cf79253d (intptr,intptr,bool,int,int,int,int)
--- End of stack trace from previous location where exception was thrown ---
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <9ed750c09b8f45beba214b7bbb241a9e>:0
at Java.Interop.JniEnvironment+InstanceMethods.CallNonvirtualVoidMethod (Java.Interop.JniObjectReference instance, Java.Interop.JniObjectReference type, Java.Interop.JniMethodInfo method, Java.Interop.JniArgumentValue* args) [0x00089] in <5d8c9bafeaf046dc9177898a7b050df7>:0
at Java.Interop.JniPeerMembers+JniInstanceMethods.InvokeNonvirtualVoidMethod (System.String encodedMember, Java.Interop.IJavaPeerable self, Java.Interop.JniArgumentValue* parameters) [0x0001f] in <5d8c9bafeaf046dc9177898a7b050df7>:0
at Android.Views.ViewGroup.Layout (System.Int32 l, System.Int32 t, System.Int32 r, System.Int32 b) [0x00062] in <a1ff8381ed444e628d5466b87b31cb12>:0
at Xamarin.Forms.Platform.Android.AppCompat.TabbedPageRenderer.OnLayout (System.Boolean changed, System.Int32 l, System.Int32 t, System.Int32 r, System.Int32 b) [0x0012d] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.FormsViewGroup.n_OnLayout_ZIIII (System.IntPtr jnienv, System.IntPtr native__this, System.Boolean p0, System.Int32 p1, System.Int32 p2, System.Int32 p3, System.Int32 p4) [0x00008] in <02fb1ab8b89c4df098f6cb3625e0c285>:0
at (wrapper dynamic-method) System.Object:d1f0d31b-ee80-4c16-b4e2-4cf698efa9e5 (intptr,intptr,bool,int,int,int,int)
--- End of stack trace from previous location where exception was thrown ---
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <9ed750c09b8f45beba214b7bbb241a9e>:0
at Java.Interop.JniEnvironment+InstanceMethods.CallNonvirtualVoidMethod (Java.Interop.JniObjectReference instance, Java.Interop.JniObjectReference type, Java.Interop.JniMethodInfo method, Java.Interop.JniArgumentValue* args) [0x00089] in <5d8c9bafeaf046dc9177898a7b050df7>:0
at Android.Runtime.JNIEnv.CallNonvirtualVoidMethod (System.IntPtr jobject, System.IntPtr jclass, System.IntPtr jmethod, Android.Runtime.JValue* parms) [0x00015] in <a1ff8381ed444e628d5466b87b31cb12>:0
at Xamarin.Forms.Platform.Android.FormsViewGroup.MeasureAndLayout (System.Int32 p0, System.Int32 p1, System.Int32 p2, System.Int32 p3, System.Int32 p4, System.Int32 p5) [0x000f6] in <02fb1ab8b89c4df098f6cb3625e0c285>:0
at Xamarin.Forms.Platform.Android.VisualElementTracker.UpdateLayout () [0x000bc] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].UpdateLayout () [0x0000e] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].OnLayout (System.Boolean changed, System.Int32 l, System.Int32 t, System.Int32 r, System.Int32 b) [0x0003d] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.AppCompat.NavigationPageRenderer.OnLayout (System.Boolean changed, System.Int32 l, System.Int32 t, System.Int32 r, System.Int32 b) [0x0000d] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.FormsViewGroup.n_OnLayout_ZIIII (System.IntPtr jnienv, System.IntPtr native__this, System.Boolean p0, System.Int32 p1, System.Int32 p2, System.Int32 p3, System.Int32 p4) [0x00008] in <02fb1ab8b89c4df098f6cb3625e0c285>:0
at (wrapper dynamic-method) System.Object:d1f0d31b-ee80-4c16-b4e2-4cf698efa9e5 (intptr,intptr,bool,int,int,int,int)
--- End of stack trace from previous location where exception was thrown ---
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <9ed750c09b8f45beba214b7bbb241a9e>:0
at Java.Interop.JniEnvironment+InstanceMethods.CallNonvirtualVoidMethod (Java.Interop.JniObjectReference instance, Java.Interop.JniObjectReference type, Java.Interop.JniMethodInfo method, Java.Interop.JniArgumentValue* args) [0x00089] in <5d8c9bafeaf046dc9177898a7b050df7>:0
at Android.Runtime.JNIEnv.CallNonvirtualVoidMethod (System.IntPtr jobject, System.IntPtr jclass, System.IntPtr jmethod, Android.Runtime.JValue* parms) [0x00015] in <a1ff8381ed444e628d5466b87b31cb12>:0
at Xamarin.Forms.Platform.Android.FormsViewGroup.MeasureAndLayout (System.Int32 p0, System.Int32 p1, System.Int32 p2, System.Int32 p3, System.Int32 p4, System.Int32 p5) [0x000f6] in <02fb1ab8b89c4df098f6cb3625e0c285>:0
at Xamarin.Forms.Platform.Android.VisualElementTracker.UpdateLayout () [0x000bc] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].UpdateLayout () [0x0000e] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.AppCompat.Platform.Xamarin.Forms.Platform.Android.IPlatformLayout.OnLayout (System.Boolean changed, System.Int32 l, System.Int32 t, System.Int32 r, System.Int32 b) [0x00022] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Xamarin.Forms.Platform.Android.PlatformRenderer.OnLayout (System.Boolean changed, System.Int32 l, System.Int32 t, System.Int32 r, System.Int32 b) [0x00018] in <e3d80a9502b84521b638fbb10cce4a8a>:0
at Android.Views.ViewGroup.n_OnLayout_ZIIII (System.IntPtr jnienv, System.IntPtr native__this, System.Boolean changed, System.Int32 l, System.Int32 t, System.Int32 r, System.Int32 b) [0x00008] in <a1ff8381ed444e628d5466b87b31cb12>:0
at (wrapper dynamic-method) System.Object:e5b29d3a-037a-43b4-bc64-ad23cf79253d (intptr,intptr,bool,int,int,int,int)
